zzre/game/systems/model/BackdropLoader.cs

```diff
@@ -52,15 +52,30 @@ private void HandleSceneLoaded(in messages.SceneLoaded message)
             : null;
         switch(dynBackdropId)
         {
-            case 10: CreateStaticBackdrop("fbgsm01p", depthTest: false, depthWrite: false,
-                rotation: Quaternion.CreateFromAxisAngle(Vector3.UnitX, MathF.PI / -2));
+            case 2: // Forest
+                CreateStaticBackdrop("ebg01h", depthTest: false, depthWrite: false);
+                break;
+            case 3: // Horizon
+                CreateStaticBackdrop("sbg01m", hasFog: false);
```

Comment:
I have rechecked horizon, it does rotate sbg01m, which can be replicated by adding `components.behaviour.Rotate(Vector3.UnitY, 1f)`